Makefile

```diff
@@ -5,28 +5,17 @@ INSTALL_DIR = ${PWD}/third_party/install
 
 all: format
 
-IN_ENV = if [ -e env/bin/activate ]; then . env/bin/activate; fi; source utils/environment.python.sh;
-env:
+IN_ENV = if [ -e env/conda/bin/activate ]; then . env/conda/bin/activate; fi;
+conda:
 	git submodule update --init --recursive
-	virtualenv --python=python3 env
-	# Install fasm from third_party
-	$(IN_ENV) pip install --upgrade -e third_party/fasm
-	# Install edalize from third_party
-	$(IN_ENV) pip install --upgrade -e third_party/edalize
+	mkdir -p env
+	source utils/conda.sh
 	# Install requirements
 	$(IN_ENV) pip install -r requirements.txt
```

Comment:
We should use an `environment.yaml` file to create the environment. See how sv-tests does it.